### PR TITLE
Trip class

### LINKED ISSUE #4: Set up Classes
- [x] Traveler Class
- [x] Trip class

### LINKED ISSUE #14: Build Trip test suite

### LINKED ISSUE #16: Set up time helper test suite

### LINKED ISSUE #17: Set up time helper functions

### PR DESCRIPTION
**Pull request by BlakeDonn**
#### What’s this PR do?  

- This PR implements a hand-built time module to help with accessing trip status.
- Adds Trip class and test suite 
- Adds Time helper script and test suite

#### Where should the reviewer start?  

- Running the test suite and evaluating how everything is working will probably be the best place to start

#### How should this be manually tested?  

- NPM test
 
#### Any background context you want to provide?  

- This choice was mad to avoid using packages like moment.js, but may reconsider this in the future.
 
#### What are the relevant tickets?  

closes #4 closes #14 closes #17 closes #16

### CODE REVIEW

File: src/classes/trip.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import time from '../scripts/time';
+
+class Trip {
+  constructor(tripInfo) {
+    this.tripLocation = tripInfo.destinyID;
+    this.tripStatus = time.isBetween(tripInfo.status, tripInfo.date.split('/'), time.daysFromDate(tripInfo.date.split('/'), tripInfo.duration))
```
Comment (BlakeDonn) [inline src/classes/trip.js]: Once again, the king of obnoxiously long lines. 

Going to probably handle the split and refine this in scripts, just have this located here temporarily

File: src/classes/trip.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import time from '../scripts/time';
+
+class Trip {
```
Comment (aemcdonald) [inline src/classes/trip.js:3]: Pretty neat way to organize your trip class. Very cool!

File: src/classes/trip.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  constructor(tripInfo, destination) {
+    this.tripLocation = tripInfo.destinyID;
+    this.tripStatus = time.isBetween(tripInfo.status, tripInfo.date.split('/'), time.daysFromDate(tripInfo.date.split('/'), tripInfo.duration));
+    this.year = tripInfo.date.split('/')[0];
+    this.destination = destination;
+  }
+}
```
Comment (aemcdonald) [inline src/classes/trip.js:10]: Not sure what your plan is but be thinking ahead about where you want to put your calculate trip cost method. I think there are strong arguments either way for it to live on Trip class or on the Destination class with the way the API data is formatted

**Review by aemcdonald — APPROVED** [review]
Looks great Blake! Your code is very concise & clean. Nice job!

**Pull request merged**